portal-api/src/v1/controllers/utilisation-report-service/previous-reports.controller.js

```diff
@@ -1,11 +1,23 @@
+const orderBy = require('lodash/orderBy');
 const api = require('../../api');
 const { getMonthName } = require('../../../utils/getMonthName');
 
+/**
+ * Returns a set of unique years based on reports from the database.
+ * @param {Object[]} reports - reports from the database
+ * @returns {Number[]} - unique set of years
```

Comment:
I think the type here would just be the primitive `number` instead of the `Number` interface (and this would probably be the case for all the types we use unless we've specifically used a Class or Interface).